app/main.py

```diff
@@ -1,6 +1,21 @@
+import os
+
+import requests
+from dotenv import load_dotenv
+
+load_dotenv()
+URL = "http://api.weatherapi.com/v1/current.json"
```

Comment:
According to the checklist, you should use constants for URLs. However, in your case, the URL is not a constant because it includes a variable part (the city name). It would be better to define a constant for the base URL and then append the variable part when making the request.